components/eventing-controller/testing/subscriber.go

```diff
@@ -78,22 +81,46 @@ func NewSubscriber() *Subscriber {
 			return
 		}
 	})
+	return mux
+}
+
+func NewSubscriber(opts ...SubscriberOption) *Subscriber {
+	mux := getServeMux()
```

Comment:
(nit) I think it'd make sense to move this line inside the if statement at 95.

Reply:
agree!